src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.IO.cs

```diff
@@ -160,17 +167,43 @@ private async Task WriteBody(bool flush = false)
                 }
             }
             // We want to swallow IO exception and allow app to finish writing
+            catch (ConnectionResetException)
+            {
+                ConnectionReset();
```

Comment:
Add a comment about how we don't want to set the exception for connection reset on writes.

Reply:
It's above the exception handler